lib/src/clojure_lsp/feature/add_missing_libspec.clj

```diff
@@ -285,29 +284,19 @@
       (find-refer-require-suggestions sym missing-requires db))))
 
 (defn add-require-suggestion [zloc chosen-ns chosen-alias chosen-refer db]
-  (let [ns-usages-nodes (edit/find-forms zloc #(when-let [sym (safe-sym %)]
-                                                 (and (= chosen-ns (namespace sym))
-                                                      ;; See note below. This is always false, except in tests
-                                                      (not= chosen-alias (namespace sym)))))
-        chosen-require  (if chosen-alias
-                          (add-known-alias zloc (symbol chosen-alias) (symbol chosen-ns) db)
-                          (add-known-refer zloc (symbol chosen-refer) (symbol chosen-ns) db))]
-    (seq
-      (concat chosen-require
-             ;; TODO: the intention of the following code appears to be to
-             ;; convert un-aliased symbols to aliased symbols, e.g.
-             ;; `clojure.string/split` -> `my-str/split`. However, when on a
-             ;; `clojure.string/split` node, the UI will only ever suggest
-             ;; 'clojure.string' as an alias. So, in practice, there's never any
-             ;; meaningful change. Proposal: delete this code and rely on
-             ;; "rename" to change aliases. Alternatively, make the UI accept
-             ;; custom aliases.
-              (when chosen-alias
```

Comment:
I've finally understood—in fact, the code actions WILL suggest renames. If you're on `clojure.string/split`, the code actions will suggest "require clojure.string :as string". If you choose that, this code will convert the usage to `string/split`. That's very common, and very useful. I think the comment is more accurate now.

Reply:
Yes, I forgot I added that some time ago 😅